contracts/cw20-merkle-airdrop/README.md

```diff
@@ -0,0 +1,98 @@
+# CW20 Merkle Airdrop
+
+This is a merkle airdrop smart contract that works with cw20 token specification Mass airdrop distributions made cheap
+and efficient.
+
+Explanation of merkle airdrop: [Medium Merkle Airdrop: the Basics](https://medium.com/smartz-blog/merkle-airdrop-the-basics-9a0857fcc930)
+
+Traditional and non-efficient airdrops:
+
+- Distributor creates a list of airdrop
+- Sends bank send messages to send tokens to recipients
+  **Or**
+- Stores list of recipients on smart contract data
+- Recipient claims the airdrop
+
+These two solutions are very ineffective when recipient list is big. First, costly because bank send cost for the
+distributor will be costly. Second, whole airdrop list stored in the state, again costly.
+
+Merkle Airdrop is very efficient even when recipient number is massive.
+
+This contract works with multiple airdrop rounds, meaning you can execute several airdrops using same instance.
+
+Uses **SHA3 Keccak 256** for merkle root tree construction.
```

Comment:
Why this choice?
To be compatible with Ethereum obviously, but do we need to?
Are there frontend tools we can reuse by doing so?

I would use sha2-256 or sha3-256 (non-keccak) or maybe blake2. Keccak is only used in Ethereum, the rest are more standard libs.

Reply:
I pulled this code base from anchor protocol(Apache 2.0) licence.
I believe developers come from solidity background.
I thought of changing it but left it for your decision. I will apply it!

Reply:
Let's leave it.